code/backtracking/src/n_queen/nqueen_bitmask.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+
+long long int n, ans = 0, DONE;
```

Comment:
n should be local to main. ans should be local to `solve` and returned. When making the changes I say, do not create a new PR. Just push to the same branch to update.